.ruby-version

```diff
@@ -1 +1 @@
-2.6.5
+2.7.2
```

Comment:
might be a good time to remove this in favour of asdf like we have for registers?